private/pkg/normalpath/normalpath_windows.go

```diff
@@ -83,10 +83,17 @@ func EqualsOrContainsPath(value string, path string, pathType PathType) bool {
 
 	// Walk up the path and compare at each directory level until there is a
 	// match or the path reaches its root (either `/` or `.`).
-	for curPath := path; !strings.EqualFold(curPath, pathRoot); curPath = Dir(curPath) {
```

Comment:
The approach itself seems logical enough to me. However, I think this means you can delete the `pathRoot` calculation above, right? Curious if/how the behavior changes with UNC paths, too. If it works for those, then this seems like a major simplification.

Reply:
So I think that this should work with UNC paths, since `Dir` should return the volume name as `VolumeName` returns it: https://cs.opensource.google/go/go/+/master:src/path/filepath/path.go;l=654
So I think you're right, we'll be able to get rid of the `pathRoot` negotiation. Let me test this to make sure!

Reply:
Just tested, seems like this works (it's good that we have test cases for this :D)